### ISSUE TITLE
Missing brushes

### ISSUE THREAD

**Issue by niels9001** [microsoft/microsoft-ui-xaml#10695]
### Describe the bug

Compared to the [Windows UI kit](https://www.figma.com/community/file/1440832812269040007/windows-ui-kit), certain brushes are missing from WinUI.

The WinUI Gallery also highlights these brushes as missing:

`ControlFillColorQuarternaryBrush`
`CardBackgroundTertiaryBrush`
`SolidBackgroundFillColorQuinaryBrush`
`SolidBackgroundFillColorSenaryBrush`

### Steps to reproduce the bug

Open the Windows UI Kit and compare it against the brushes that ship with WinUI,

### NuGet package version

None

**Comment by niels9001** [comment]
@Saharsh979
